cli/flags.rs

```diff
@@ -1863,8 +1875,15 @@ fn coverage_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
     None => vec![],
   };
   let lcov = matches.is_present("lcov");
+  let output = if let Some(output) = matches.value_of("output") {
+    flags.allow_write = Some(vec![]);
```

Comment:
Uhoh, this is a red flag. Why is this flag being set here?

Reply:
sorry, this was mistake, I have removed this !!